src/styles/variables.scss

```diff
@@ -0,0 +1,77 @@
+@use "sass:map";
+
+$dark: #2B2C34;
+$dark-grey: #9A9494;
+$grey: #C0C0C0;
+$light-grey: #E4E5E9;
+$extra-light: #EFF0F3;
+$white: #FFFFFE;
+$blue: #6246EA;
+$border-dark: rgba($dark, 0.88);
+
+.alert {
+  border: 1px solid $border-dark;
+}
```

Comment:
Will you use this class often?
If not, can you set it directly in the component as it's only one line?